src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -233,10 +234,16 @@ public boolean verifyTitle(String title) {
     return true;
   }
 
-  public void verifyURLcontains(String givenString) {
+  public boolean isStringInURL(String exptectedString) {
```

Comment:
`isStringInURL` --> `expectedString`, doesn't have sense.
try this:
`isStringInURL` --> `substring`